kattis2canvas.py

```diff
@@ -497,5 +519,28 @@ def get_best_submissions(offering: str, assignment_id: str) -> {str: {str: Submi
     return best_submissions
 
 
+@top.command()
+@click.argument("canvas_course")
+def sendemail(canvas_course):
```

Comment:
i love this command so much!

you can use canvas.create_conversation([[link.canvas_user.id], body, subject=..., group_conversation=True) instead of the web API

Reply:
Thank you professor, updated to python api.